internal/bminventory/inventory_test.go

```diff
@@ -123,11 +123,13 @@ func mockClusterRegisterSteps() {
 	mockVersions.EXPECT().GetReleaseImage(gomock.Any(), gomock.Any()).Return(common.TestDefaultConfig.ReleaseImage, nil).Times(1)
 	mockOperatorManager.EXPECT().GetSupportedOperatorsByType(models.OperatorTypeBuiltin).Return([]*models.MonitoredOperator{&common.TestDefaultConfig.MonitoredOperator}).Times(1)
 	mockProviderRegistry.EXPECT().SetPlatformUsages(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(nil)
+	mockProviderRegistry.EXPECT().GetActualSchedulableMasters(gomock.Any()).AnyTimes()
```

Comment:
What about testing bad flows?

Reply:
Unit tests are not implemented yet, I will do it in my next push, hopefully

Reply:
Still relevant no?

Reply:
done